src/SimpleFileUpload.tsx

```diff
@@ -1,28 +1,32 @@
 import * as React from 'react';
 import { FieldProps, getIn } from 'formik';
 import FormControl from '@material-ui/core/FormControl';
-import InputLabel from '@material-ui/core/InputLabel';
-import Input from '@material-ui/core/Input';
+import InputLabel, { InputLabelProps } from '@material-ui/core/InputLabel';
+import Input, { InputProps } from '@material-ui/core/Input';
 import FormHelperText from '@material-ui/core/FormHelperText';
 
 export interface SimpleFileUploadProps extends FieldProps {
   label: string;
   disabled?: boolean;
+  inputProps: InputProps;
```

Comment:
Should we not make this match the existing api? Taken from https://material-ui.com/api/text-field/

![image](https://user-images.githubusercontent.com/3578740/58765474-dc950e00-8573-11e9-974a-d73ed4f0224b.png)

Reply:
Agreed. Change made. I made a mistaken assumption earlier and thought it needed to be named in `camelCase`, but after looking into the error that tslint was throwing me, realized I could rename the prop within the component so that the public API was consistent with material-ui